app/(routes)/about/faqs/page.tsx

```diff
@@ -0,0 +1,37 @@
+import Button from "@/components/shared/Button"
+
+export default function AboutFaQs() {
+  return (
+    <div className="px-7">
+      <h4 className="text-xl pb-4 font-bold leading-none tracking-tight text-gray-900">
+        Frequently Asked Questions
+      </h4>
+      <h5 className="text-base font-bold leading-none tracking-tight text-gray-900">
+        How can I get involved?
+      </h5>
+        <p>
+          If you wish to volunteer to Boston Code and Coffee lease reach out to us on bostoncodeandcoffee@gmail.com and we will set up an initial call to discuss your participation. 
```

Comment:
Missing 'p' on line 13 in "If you wish to volunteer to Boston Code and Coffee lease..."